tests/testthat/test-append_round.R

```diff
@@ -0,0 +1,68 @@
+test_that("append_round works", {
+  config <- hubUtils::read_config_file(
+    config_path = test_path("testdata/tasks-append.json")
+  ) |> as_config()
+  round <- create_new_round()
+
+  # Adding one round works
+  append_round(config, round)
+
+  # Adding two round works
+  add_2_rounds <- append_round(
+    config = config, round,
+    create_new_round("2024-09-25")
+  )
+
+  expect_length(add_2_rounds$rounds, 3)
+  expect_s3_class(add_2_rounds, c("config", "list"))
+})
+
```

Comment:
Here's the test for the version of the function that updates the schema URL.

```suggestion

test_that("append_round allows for and updates old schema url", {
  config <- hubUtils::read_config_file(
    config_path = test_path("testdata/tasks-append.json")
  ) |> as_config()
  round <- create_new_round()
  ver <- hubUtils::extract_schema_version(attr(config, "schema_id"))
  cfg <- basename(attr(config, "schema_id"))
  ver <- file.path(ver, cfg)
  prefix <- "https://raw.githubusercontent.com/"
  owner <- "Infectious-Disease-Modeling-Hubs" 
  suffix <- "/schemas/main/"
  attr(config, "schema_id") <- paste0(prefix, owner, suffix, ver)
 
  # Adding one round works
  add_round <- append_round(config, round)
  expect_length(add_round$rounds, 2)
  expect_s3_class(add_round, c("config", "list"))
  expected <- paste(prefix, "hubverse-org", suffix, ver)
  expect_equal(attr(add_round, "schema_id"), expected)
})

```

Reply:
resolving this in favor of Anna's suggestion